template.yaml

```diff
@@ -574,6 +572,39 @@ Resources:
   DynamodbEventFanoutStreamRecordsDLQ:
     Type: "AWS::SQS::Queue"
 
+  UpdateDoiStatusHandlerFunction:
+    Type: AWS::Serverless::Function
+    Properties:
+      Layers:
+        - !Ref NvaPublicationCommonsLayer
+      CodeUri: update-doi-status
+      Handler: no.unit.nva.doi.handler.UpdateDoiStatusHandler::handleRequest
+      Runtime: java11
+      MemorySize: 1408
+      Environment:
+        Variables:
+          EVENT_BUS: !Ref EventBusName
+          AWC_ACCOUNT_ID: !Ref AWS::AccountId
+      Events:
+        EventBridgeEvent:
+          Type: EventBridgeRule
+          Properties:
+            EventBusName: !Ref EventBusName
+            Pattern:
+              detail:
+                responsePayload:
+                  type:
+                    - doi.updateDoiStatus
+      EventInvokeConfig:
+        DestinationConfig:
+          OnSuccess:
+            Type: EventBridge
+            Destination: !Ref EventBusArn
+          OnFailure:
+            Type: SQS
+            Destination: !Ref DlqArn
```

Comment:
Is it more correct to have separate DLQs for each function?

Reply:
I do not know. Happy to get input, I just put them under the same DLQ for now.

Reply:
My gut feeling is that we do, should probably do a team decision on this.

Reply:
https://github.com/BIBSYSDEV/nva-publication-api/pull/97#issuecomment-732875369 (now I found the place to add the comment, github UI is weird sometimes)